deepchem/feat/graph_features.py

```diff
@@ -630,6 +630,22 @@ class ConvMolFeaturizer(MolecularFeaturizer):
   Duvenaud graph convolutions [1]_ construct a vector of descriptors for each
   atom in a molecule. The featurizer computes that vector of local descriptors.
 
+  Examples
+  ---------
+  Using ConvMolFeaturizer to create featurized fragments derived from molecules of interest.
+  This is used only in the context of performing interpretation of models using atomic
+  contributions (atom-based model interpretation)
```

Comment:
Giving an example of only the special purpose usage (atom depleted fragments) but no example of the typical usage could be confusing.  How about first showing the normal usage, and then a second example with `per_atom_fragmentation=True`?

Reply:
I agree))) there was no example at all before, so I added only one concerned with my feature..i can extend it